js/utils.js

```diff
@@ -0,0 +1,13 @@
+/** 이메일 정규식 검사 */
+const isValidEmail = (v) => /^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(v.trim());
+
+/** 최소 글자수 8 */
+const isMin8 = (v) => v.trim().length >= 8;
+
+/** 공백 확인 */
+const isRequired = (v) => v.trim().length > 0;
+
+/** 문자열 일치 확인 */
+const isSame = (a, b) => String(a) === String(b);
```

Comment:
💊 제안
isSame이라는 함수명은 변수처럼 느껴지고, 무엇을 비교하는지 명확하지 않습니다. 함수인지 변수인지, 어떤 값을 비교하는 함수인지 드러나는 이름으로 수정해주시면 더 이해하기 쉽습니다.

참고만 해주세요~

```suggestion
const isSameString = (a, b) => String(a) === String(b);
```

함수명이 isSameString이 되면서 string 비교를 의도한 것임이 명확히 드러나고, 내부에서 String()으로 형변환해주는 이유도 설명이 잘 됩니다.